platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/UserPropertiesDeletionScheduler.java

```diff
@@ -0,0 +1,165 @@
+    deletionScheduled = false;
+  }
+
+  public synchronized void scheduleDeletion() {
+    if (!deletionScheduled && USERS_PROPERTIES_FILE_PATH.toFile().exists()) {
+      String cron = getCron();
+      if (cron == null) {
+        return;
+      }
+      try {
+        CamelContext context = new DefaultCamelContext();
+        RouteBuilder builder =
+            new RouteBuilder() {
+              @Override
+              public void configure() throws Exception {
+                fromF("quartz2://scheduler/deletionTimer?cron=%s", cron)
+                    .bean(UserPropertiesDeletionScheduler.class, "deleteFiles");
+              }
+            };
+
+        context.addRoutes(builder);
+        context.start();
+        deletionScheduled = true;
+      } catch (Exception e) {
+        LOGGER.debug("Unable to create route", e);
+      }
+    }
+  }
+
+  @VisibleForTesting
+  String getCron() {
+    if (!TEMP_TIMESTAMP_FILE_PATH.toFile().exists()) {
+
+      // Create temp file and add timestamp
+      Instant instant = Instant.now();
+      if (writeTimestamp(instant)) {
+        return null;
+      }
+      return cronCalculator(instant);
+
+    } else {
+
+      // Read timestamp
+      Instant firstInstall = installationDate();
+      if (firstInstall == null) {
+        return null;
+      }
+
+      if (Instant.now().isAfter(firstInstall.plus(Duration.ofDays(3)))) { // Timestamp expired
+        deleteFiles();
+        return null;
+      }
+
+      return cronCalculator(firstInstall);
+    }
+  }
+
+  private boolean writeTimestamp(Instant instant) {
+    try {
+      if (!TEMP_TIMESTAMP_FILE_PATH.toFile().createNewFile()) {
+        LOGGER.debug("Unable to create a temporary file.");
+        return true;
+      }
+
+      ObjectOutputStream objectOutputStream =
+          new ObjectOutputStream(new FileOutputStream(TEMP_TIMESTAMP_FILE_PATH.toFile()));
+      objectOutputStream.writeObject(instant);
+      objectOutputStream.close();
+
+    } catch (IOException e) {
+      LOGGER.debug("Unable to access the temporary file", e);
+      return true;
+    }
+    return false;
+  }
+
+  public void deleteFiles() {
+    try {
+      Files.deleteIfExists(USERS_PROPERTIES_FILE_PATH);
+      Files.deleteIfExists(TEMP_TIMESTAMP_FILE_PATH);
+      deletionScheduled = false;
+    } catch (IOException e) {
+      LOGGER.debug("Unable to delete file", e);
+    }
+  }
+
+  @VisibleForTesting
+  String cronCalculator(Instant firstInstall) {
+    Instant threeDayTimestamp = firstInstall.plus(Duration.ofDays(3));
+    LocalDateTime localDateTime =
+        LocalDateTime.ofInstant(threeDayTimestamp, ZoneId.systemDefault());
+
+    return String.format(
+            "0 %d %d %d %d ? %d",
+            localDateTime.getMinute(),
+            localDateTime.getHour(),
+            localDateTime.getDayOfMonth(),
+            localDateTime.getMonthValue(),
+            localDateTime.getYear())
+        .replaceAll(" ", "+");
```

Comment:
❓ Why this replace instead of just writing your format string with `+` instead of spaces?